spec/editors_draft.html

```diff
@@ -3298,6 +3298,231 @@ <h2>Evaluation Semantics</h2>
 
   </section>
 
+  <section id="import-export-composite-values">
+    <h1>Importing and Exporting of Composite Values</h1>
+
+    <section class="informative">
+      <h2>Motivation</h2>
+      <p>
+        In the normal handling of an RDF syntax, a blank node is scoped to the document, and any identifiers used in that document are merely a syntactic convenience for referencing blank nodes freshly allocated during parsing.
+        For example,
+      </p>
+        
+      <pre id="load-renaming-blank-example"
+         data-transform="updateExample"
+         data-content-type="text/turtle"
+         class="nohighlight example">
+      <!--
+      _:b1 ex:p4 "[ <http://example.org/alice>, <http://example.org/bob>, _:eve ]"^^cdt:List .
+      -->
+      </pre>
+      
+      <p>
+        Parsing this data as Turtle will result in a single triple whose subject is a blank node (now dissociated from the `b1` identifier), and whose object is a three-element <a>cdt:List literal</a>.
+        However, using a Turtle parser not aware of Composite Values will result in an object literal containing the substring "_:eve".
```

Comment:
I think this has to be at least in part a job for the parsers, because it depends on the parser's internal state for mapping blank node identifiers to fresh blank nodes. Most parsers don't expose that map publicly, so our proposed rewriting process has to be handled during parsing (so that we can align bnode identifiers inside composite literals with a lexical value that will map to the same blank node [via lexical-to-value mapping] as is produced by the parser when seeing that same identifier outside of the composite literal values).

Reply:
So, your point with this sentence is actually not about whether an RDF processor, in general, is aware of our composite values, but indeed only about whether the Turtle parser of such a processor is aware of composite values?

If yes, that was not directly clear to me from the sentence.

Reply:
I think this is difficult text to figure out because as implementors, we may want to distinguish "RDF parser" from "RDF processor" or "SPARQL system". But doesn't our requirement here impose a tighter coupling between a parser and the SPARQL system than is often found in real systems? That is, the parser has to be aware of the lexical-to-value mapping that the system will use, whereas most RDF parsers just emit triples in some structured form, not caring about how the lexical forms in the emitted literals get mapped to the value space during processing.

Again, let me think on how to improve this language to get closer to the intention…

Reply:
The new text that you wrote for this part in your recent commits nicely addresses my concern!